### PR TITLE
wsman:  sblim-sfcc, openwsman, wsmancli

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

File: pkgs/development/libraries/openwsman/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name = "openwsman-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/Openwsman/openwsman/archive/v${version}.tar.gz";
+    sha256 = "0gw2dsjxzpchg3s85kplwgp9xhd9l7q4fh37iy7r203pvir4k6s4";
+  };
+
+  buildInputs = [ autoconf automake libtool pkgconfig libxml2 curl cmake pam sblim-sfcc ];
+
+  configurePhase = ''mkdir build
```
Comment (ttuegel) [inline pkgs/development/libraries/openwsman/default.nix]: It would be better to use the CMake `setupHook` for this.

Comment (ttuegel) [inline pkgs/development/libraries/openwsman/default.nix]: So, you could replace this `configurePhase` with:
```
cmakeFlags = [
  "-DCMAKE_BUILD_RUBY_GEM=no"
];
preConfigure = ''
  cmakeFlags="$cmakeFlags $(uname -m)";
'';
```

Comment (deepfire) [inline pkgs/development/libraries/openwsman/default.nix]: @ttuegel, replacing things like this fails this way:
```
[deepfire@andromedae:~/src/nixpkgs]$ nsr /nix/store/hhvzj4p1bc3pqk8wn522s2lzad4lg447-openwsman-2.6.0.drv
these derivations will be built:
  /nix/store/hhvzj4p1bc3pqk8wn522s2lzad4lg447-openwsman-2.6.0.drv
building path(s) ‘/nix/store/8kk3kv938m91mc9swkhnw6p9g7p4pnwh-openwsman-2.6.0’
unpacking sources
unpacking source archive /nix/store/hglw6qfi98s087m3g3f664w0rfdkbcd9-v2.6.0.tar.gz
source root is openwsman-2.6.0
patching sources
configuring
fixing cmake files...
cmake flags: -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DCMAKE_INSTALL_NAME_DIR=/nix/store/8kk3kv938m91mc9swkhnw6p9g7p4pnwh-openwsman-2.6.0/lib -DCMAKE_INSTALL_PREFIX=/nix/store/8kk3kv938m91mc9swkhnw6p9g7p4pnwh-openwsman-2.6.0 -DCMAKE_BUILD_RUBY_GEM=no x86_64 
CMake Error: The source directory "/tmp/nix-build-openwsman-2.6.0.drv-0/openwsman-2.6.0/build/x86_64" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
builder for ‘/nix/store/hhvzj4p1bc3pqk8wn522s2lzad4lg447-openwsman-2.6.0.drv’ failed with exit code 1
```

Comment (ttuegel) [inline pkgs/development/libraries/openwsman/default.nix]: @deepfire D'oh, that should've been
```
preConfigure = ''
  cmakeFlags="$cmakeFlags -DPACKAGE_ARCHITECTURE=$(uname -m)";
'';
```

**Comment by ttuegel** [comment]
Do you want to add yourself as maintainer? It just means you'll get e-mailed if the package breaks, and people will usually ping you on GitHub if they want to update the package.

**Pull request merged**